Cargo.toml

```diff
@@ -50,24 +44,19 @@ render = [
 
 # Optional bevy crates
 bevy_audio = ["bevy_internal/bevy_audio"]
+bevy_core_pipeline = ["bevy_internal/bevy_core_pipeline"]
+bevy_render = ["bevy_internal/bevy_render"]
+bevy_text = ["bevy_internal/bevy_text"]
+bevy_pbr = ["bevy_internal/bevy_pbr"]
+bevy_sprite = ["bevy_internal/bevy_sprite"]
 bevy_dynamic_plugin = ["bevy_internal/bevy_dynamic_plugin"]
 bevy_gilrs = ["bevy_internal/bevy_gilrs"]
 bevy_gltf = ["bevy_internal/bevy_gltf"]
-bevy_wgpu = ["bevy_internal/bevy_wgpu"]
 bevy_winit = ["bevy_internal/bevy_winit"]
 
-bevy_core_pipeline = ["bevy_internal/bevy_core_pipeline"]
-bevy_render2 = ["bevy_internal/bevy_render2"]
-bevy_sprite2 = ["bevy_internal/bevy_sprite2"]
-bevy_pbr2 = ["bevy_internal/bevy_pbr2"]
-bevy_gltf2 = ["bevy_internal/bevy_gltf2"]
-bevy_ui2 = ["bevy_internal/bevy_ui2"]
-bevy_text2 = ["bevy_internal/bevy_text2"]
-
 trace_chrome = ["bevy_internal/trace_chrome"]
 trace_tracy = ["bevy_internal/trace_tracy"]
 trace = ["bevy_internal/trace"]
-wgpu_trace = ["bevy_internal/wgpu_trace"]
```

Comment:
We will need `wgpu_trace` for capturing traces to attach to bug reports against wgpu so this should be proxied through to all crates using wgpu, right?

Reply:
Hmm yeah good point.